app/models/agent_bot_inbox.rb

```diff
@@ -3,7 +3,7 @@
 # Table name: agent_bot_inboxes
 #
 #  id           :bigint           not null, primary key
-#  status       :integer          default("active")
```

Comment:
Why is this changed?

Reply:
Someone forgot to commit this. I committed myself.